app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: t('flash.books.create') }
+        format.html { redirect_to book_url(@book), notice: t('flash.create', model: t('activerecord.models.book')) }
```

Comment:
```suggestion
        format.html { redirect_to book_url(@book), notice: t('flash.create', model: Book.model_name.human) }
```

これでも同じ結果になります。
キー名を文字列で指定するとうっかりtypoが発生したりするのでメソッドを使った方がベターです。

Reply:
確認いただきありがとうございました！